backends/views.py

```diff
@@ -24,4 +26,13 @@ def complete(request, *args, **kwargs):
         request.session[key] = backend_state
 
     # Continue with social_core pipeline
-    return social_complete(request, *args, **kwargs)
+    social_complete_rtn = social_complete(request, *args, **kwargs)
```

Comment:
I think this should be within the `if request.user.is_authenticated():` block, it wouldn't make sense to run it on anonymous users